### PR TITLE
Joe add winner label

### PR DESCRIPTION
**Pull request by joeShuff**

### CODE REVIEW

File: core/src/main/java/me/lihq/game/ScenarioBuilder.java [hunk 1]
```diff
@@ -119,6 +120,9 @@ Set the first clues in the list to red herrings (the number of red herrings
 
             if (randHidingSpot != null) {
                 room.addClue(clue.setTileCoordinates(randHidingSpot));
+            } else
+            {
+                System.out.println("Failed to place Clue " + clue.getName() + " in room " + room.getName());
```
Comment (beno11224) [inline core/src/main/java/me/lihq/game/ScenarioBuilder.java]: was this bit for debugging?

File: core/src/main/java/me/lihq/game/people/Player.java [hunk 2]
```diff
@@ -227,9 +227,11 @@ private void checkForExtraScore() {
             score += extra;
             if (extra == 0) {
                  game.screenManager.navigationScreen.speechboxMngr.addSpeechBox( new SpeechBox("There appear to be no extra points here, the cash pile before you is fake!"));
+                game.currentSnapshot.player.canMove = false;
             } else {
                 SpeechBox speechBox = new SpeechBox("You gained " + extra + " extra points! Lucky you :D");
                 speechBox.setTrigger(true);
+                game.currentSnapshot.player.canMove = false;
```
Comment (beno11224) [inline core/src/main/java/me/lihq/game/people/Player.java]: I assume the player.canmove is set to true somewhere else?

Comment (joeShuff) [inline core/src/main/java/me/lihq/game/people/Player.java]: Yes, when they close the speechbox it gets set to true.

Comment (beno11224) [inline core/src/main/java/me/lihq/game/people/Player.java]: cool, just wanted to check :)

File: core/src/main/java/me/lihq/game/ScenarioBuilder.java (hunk 1 above)
Comment (joeShuff) [inline core/src/main/java/me/lihq/game/ScenarioBuilder.java]: It was yes but it should now never be called as I removed the Secret Room from the room list. But its  good to keep it in for future debugging. I can remove it if everyone wants me to

Comment (beno11224) [inline core/src/main/java/me/lihq/game/ScenarioBuilder.java]: cool, no just wanted to make sure it wasnt for popping up in the game or anything

Comment (jasonmash) [inline core/src/main/java/me/lihq/game/ScenarioBuilder.java]: I suggest this gets removed - if it's not ever being run there's no reason to keep it around. It won't be too hard to add back in for future debugging (if that ever happens)

File: core/src/main/java/me/lihq/game/people/Player.java (hunk 2 above)
Comment (jasonmash) [inline core/src/main/java/me/lihq/game/people/Player.java]: Should this line ` game.currentSnapshot.player.canMove = false;` be within both cases of the if statement? Could probs move it out of the if brackets as that code is run every time, and doesn't depend on extra==0

File: core/src/main/java/me/lihq/game/ScenarioBuilder.java (hunk 1 above)
Comment (Brookke) [inline core/src/main/java/me/lihq/game/ScenarioBuilder.java]: If this is crucial to the game I would suggest an exception is raised

Comment (joeShuff) [inline core/src/main/java/me/lihq/game/ScenarioBuilder.java]: It is quite crucial, cause then a clue isnt placed in the map, could be the murder weapon. Therefore the game isn't completable. I'll change it to an exception

File: core/src/main/java/me/lihq/game/people/Player.java (hunk 2 above)
Comment (joeShuff) [inline core/src/main/java/me/lihq/game/people/Player.java]: True. Didn't notice that.

File: core/src/main/java/me/lihq/game/screen/WinScreen.java [hunk 3]
```diff
@@ -85,9 +86,32 @@ public WinScreen(GameMain game) {
...
+                   else if (snapshot.player.getPlayTime() < highestScore.player.getPlayTime()) {
+                       highestScore = snapshot;
+                   }
+            }
+        }
+
         Label title = UIHelpers.createLabel("You Found the Killer!", true);
         stage.addActor(title);
 
+        Label winnerName = UIHelpers.createLabel("The Winner is " + game.currentSnapshot.player.getName(), true);
```
Comment (joeShuff) [inline core/src/main/java/me/lihq/game/screen/WinScreen.java]: Haven't changed these to use the highest scoring player yet. Woops

**Pull request merged**